### PR TITLE
Add Support to Swift 4.1

### PR DESCRIPTION
**Pull request by mendesbarreto**

### CODE REVIEW

**Comment by mendesbarreto** [comment]
Closes issue #61

**Comment by ThomasLeblond** [comment]
Please can you merge the pull request.

**Comment by freak4pc** [comment]
Hey @ThomasLeblond. I'm gonna give the original author of the repo a chance by tomorrow and merge otherwise. 

cc/ @devxoul

File: RxKeyboard.podspec [hunk 1]
```diff
@@ -11,8 +11,8 @@ Pod::Spec.new do |s|
   s.frameworks       = 'UIKit', 'Foundation'
   s.requires_arc     = true
 
-  s.dependency 'RxSwift', '>= 4.0.0'
-  s.dependency 'RxCocoa', '>= 4.0.0'
+  s.dependency 'RxSwift', '>= 4.1.2'
+  s.dependency 'RxCocoa', '>= 4.1.2'
```
Comment (devxoul) [inline RxKeyboard.podspec]: Do we need RxSwift 4.1.2 for Swift 4.1? RxSwift 4.0.0 seems to be still work with Swift 4.1. Not all of RxKeyboard users might have upgraded RxSwift dependency to the latest, so I think we would better keep this dependency if not necessary.

**Comment by devxoul** [comment]
Hey folks, sorry for delay. I had very busy days recently. I left a single comment and would be glad if you can review it.

PS: I really appreciate to @freak4pc to follow-up these issues :pray:

File: RxKeyboard.podspec (hunk 1 above)
Comment (mendesbarreto) [reply RxKeyboard.podspec]: I think not ,

No problem man, I will change it! 

Thank you!

File: RxKeyboard.podspec [hunk 2]
```diff
@@ -11,8 +11,8 @@ Pod::Spec.new do |s|
   s.frameworks       = 'UIKit', 'Foundation'
   s.requires_arc     = true
 
-  s.dependency 'RxSwift', '>= 4.1.2'
-  s.dependency 'RxCocoa', '>= 4.1.2'
+  s.dependency 'RxSwift', '>= 4.0.0'
```
Comment (freak4pc) [inline RxKeyboard.podspec]: I think this should be `RxSwift`, '~> 4` or it won't catch 4.1.

Comment (mendesbarreto) [reply RxKeyboard.podspec]: Get it! 

I thought the signal >= would take greater than or equal to 4.0.0

Comment (freak4pc) [reply RxKeyboard.podspec]: I might be wrong actually. Not on my computer, will check later

Comment (mendesbarreto) [reply RxKeyboard.podspec]: No problem @freak4pc. I think the sinal ~> work as well

Comment (freak4pc) [reply RxKeyboard.podspec]: Ok, so the existing code should be fine

**Comment by mendesbarreto** [comment]
@freak4pc Sorry for the junk generated on my commits, but I really tried to be transparent with the work and maybe the single commits of each work I have done was a overkill 😞 . 

If you have some post ou blog to indicate to me read and improve this skill and make the next contribution better, I will be really glad to you.

**Comment by devxoul** [comment]
Thanks for your work, I've released [0.8.2](https://github.com/RxSwiftCommunity/RxKeyboard/releases/tag/0.8.2) :tada:

**Comment by ThomasLeblond** [comment]
Thanks guys, this is very helpful for us :)

**Pull request merged**